COLLADAStreamWriter/src/COLLADASWAsset.cpp

```diff
@@ -147,6 +119,10 @@ namespace COLLADASW
         default:
             mSW->appendTextElement ( CSWC::CSW_ELEMENT_UP_AXIS, CSWC::CSW_TEXT_Z_UP );
         }
+
+		addExtra();
```

Comment:
@RemiArnaud  As we discussed, It's invalid for <asset> to have <extra> element in 1.4.1. This change should be reverted and redesigned.